backend/model/dbStartup.php.smp

```diff
@@ -0,0 +1,14 @@
+<?php
+  $dsn = 'mysql:host=localhost;dbname=';
+  $username ='';
+  $password = '';
```

Comment:
Yeah. That would be bad. Wanted to include a sample config file though for open source reasons.